### PR TITLE
Make Feedback Bucketed

### PR DESCRIPTION
**Pull request by kaidamasaki**

### CODE REVIEW

**Comment by kaidamasaki** [comment]
TODO: Actually style things.

**Comment by kaidamasaki** [comment]
TODO: Validate counts

File: app/controllers/students_controller.rb [hunk 1]
```diff
@@ -19,7 +21,10 @@ def rankings
     Ranking.transaction do
       params[:rankings].each(&:permit!).map(&:to_h).each do |ranking|
         company_id = ranking[:company_id]
-        rank = ranking[:rank]
+        puts "v" * 80
+        puts ranking
+        puts "^" * 80
```
Comment (droberts-ada) [inline app/controllers/students_controller.rb]: Should this debugging output be taken out?

File: app/controllers/students_controller.rb [hunk 2]
```diff
@@ -19,7 +21,10 @@ def rankings
...
+        rank = BUCKETS[ranking[:rank]]
```
Comment (droberts-ada) [inline app/controllers/students_controller.rb]: I don't know if we have any controller testing at all for this app, but I would love to see a test that verifies that form data will be correctly translated into buckets.

**Pull request merged**